client/src/components/Workflow/Editor/Attributes.vue

```diff
@@ -135,7 +135,7 @@ export default {
                 const current_wf = this.versions[i];
                 let update_time;
                 if (current_wf.update_time) {
-                    update_time = `${format(Date.parse(current_wf.update_time), "MMM do yyyy")}, `;
+                    update_time = `${format(parseISO(current_wf.update_time, 'yyyy-MM-dd', new Date()), "MMM do yyyy")}, `; //TODO Check Time/Milliseconds; Replace with constants for Localization
```

Comment:
This looks correct to me, I would remove the comment since seconds and milliseconds are not needed here. For localization we could discuss it in a follow-up. The only thing I would add to this PR is a test to `Attributes.test.js` if possible.